From 8f0cfce3337c5ae49a6004d058dcf8b4b2ef4861 Mon Sep 17 00:00:00 2001
From: Lorenzo Pirritano <lpirritano@anaconda.com>
Date: Mon, 10 Feb 2025 13:51:31 +0100
Subject: [PATCH] fix bokeh axis visible to False

---
 splot/_viz_bokeh.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/splot/_viz_bokeh.py b/splot/_viz_bokeh.py
index 50ac5bb..8e72c23 100644
--- a/splot/_viz_bokeh.py
+++ b/splot/_viz_bokeh.py
@@ -150,7 +150,7 @@ def _plot_choropleth_fig(geo_source, attribute, bin_labels, bounds,
     # change layout
     fig.xgrid.grid_line_color = None
     fig.ygrid.grid_line_color = None
-    fig.axis.visible = None
+    fig.axis.visible = False
     return fig
 
 
@@ -261,7 +261,7 @@ def _lisa_cluster_fig(geo_source, moran_loc, cluster_labels, colors5,
     # change layout
     fig.xgrid.grid_line_color = None
     fig.ygrid.grid_line_color = None
-    fig.axis.visible = None
+    fig.axis.visible = False
     return fig
 
 
-- 
2.39.1

